components/desktop/mate/mate-control-center/Makefile

```diff
@@ -0,0 +1,71 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Alexander Pyhalov
+#
+
+include ../../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		mate-control-center
+COMPONENT_VERSION=	1.14.0
+COMPONENT_PROJECT_URL=	http://www.mate-desktop.org
+COMPONENT_SUMMARY=	MATE desktop wide preference configuration tools
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:ec29a37d287188c234badb846e43cd8e0b70e4a538684fc2a3588c9f1cea0c8e
+COMPONENT_ARCHIVE_URL=	http://pub.mate-desktop.org/releases/1.14/$(COMPONENT_ARCHIVE)
+COMPONENT_FMRI=		desktop/mate/control-center
+COMPONENT_CLASSIFICATION=	Applications/Configuration and Preferences
+COMPONENT_LICENSE= GPLv2
+COMPONENT_LICENSE_FILE= COPYING
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+PATH=/usr/gnu/bin:/usr/bin
+
+CONFIGURE_OPTIONS+=	--sysconfdir=/etc
+CONFIGURE_OPTIONS+=	--libexecdir=/usr/lib
+CONFIGURE_OPTIONS+=	--disable-static
```

Comment:
I'd prefer /usr/lib/mate to avoid clashes with gnome, but preserve everything in one place. Here it just not used.